1.0/docs/devguide/data-binding.md

```diff
@@ -451,38 +461,92 @@ as part of the element's API, or use it elsewhere in the element.
 Example:
 
     <dom-module id="x-custom">
+
       <template>
         My name is <span>{%raw%}{{computeFullName(first, last)}}{%endraw%}</span>
       </template>
+
+      <script>
+        Polymer({
+          is: 'x-custom',
+          properties: {
+            first: String,
+            last: String       
+          },
+          computeFullName: function(first, last) {
+            return first + ' ' + last;
+          }
+          ...
+        });
+      </script>
+
     <dom-module id="x-custom">
 
-    <script>
-      Polymer({
-        is: 'x-custom',
-        properties: {
-          first: String,
-          last: String       
-        },
-        computeFullName: function(first, last) {
-          return first + ' ' + last;
-        }
-        ...
-      });
-    </script>
 
 In this case, the span's `textContent` property is bound to the return value 
 of `computeFullName`, which is recalculated whenever `first` or `last` changes.
 
-**Note:** The computed binding is not called until all dependent properties are defined 
-(`!=undefined`). So each dependent properties should have a 
+### Dependent properties in computed bindings
+
+Arguments to computing functions may be _dependent properties_, which include
+any of argument types supported by the `observers` object:
+
+*   simple properties on the current scope
+*   [paths](#path-observation)
```

Comment:
These 3 links are broken. Should they point to https://www.polymer-project.org/1.0/docs/devguide/properties.html#observing-path-changes, https://www.polymer-project.org/1.0/docs/devguide/properties.html#deep-observation, and https://www.polymer-project.org/1.0/docs/devguide/properties.html#array-observation instead?